#request
aslam_rosbot/AvailablePaths paths
int32 fov_angle_deg
int32 kind_cost
bool only_last
---
#response
aslam_rosbot/SinglePath path



